.github/workflows/auto-docs.yml

```
@@ -0,0 +1,29 @@
+        run: bun run scripts/post-merge-review-check.js
+      - name: Validate docs front-matter
+        run: bun run scripts/scan-doc-frontmatter.js
+      - name: Commit & push docs if changed
+        run: |
+          git config user.name "github-actions[bot]"
+          git config user.email "github-actions[bot]@users.noreply.github.com"
+          if [ -n "$(git status --porcelain)" ]; then
+            git add docs/CHANGELOG.md tests/bug-reports || true
+            git commit -m "docs: auto-update changelog & coderabbit follow-ups [skip ci]"
+            git push
+          fi 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)